src/common/applicationDetails/__tests__/ApplicationDetails.test.tsx

```diff
@@ -130,14 +131,19 @@ describe('ApplicationDetails', () => {
   });
 
   it('renders delete lease button if handleDeleteLease prop is provided', () => {
+    // Using a component with react-modal. Silence error output.
+    jest.spyOn(console, 'error').mockImplementation(() => {
+      /* NO-OP */
+    });
+
```

Comment:
Would `ReactModal.setAppElement('body');` work here?

Reply:
Great, this works! Done.